.prettierignore

```diff
@@ -0,0 +1,3 @@
+*.json
+node_modules
+frontend/
```

Comment:
```suggestion
frontend/

```

Recomenda-se deixar uma linha em branco ao final de cada arquivo, para que ferramentas como o GitHub não exibam o ícone de `No newline at end of file`.

Isso é uma convenção e boa prática a ser seguida.